Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.14.0
+FROM alpine:3.14.1
```

Comment:
I think we should use `quay.io/giantswarm/alpine:3.14.1` instead, due to the strict limits on docker.io.